docs/app/app.component.ts

```diff
@@ -33,6 +109,14 @@ export class AppComponent implements OnInit {
             }),
             filter(data => data.route.outlet === 'primary')
         ).subscribe(data => {
+
+            if (data.route.routeConfig.path === 'organization-chart' || data.route.routeConfig.path ===  'partition-map' || data.route.routeConfig.path ===  'sankey-chart') {
```

Comment:
Perhaps it would be safer to use the custom route data property like in the Micro Focus site?

```
{ path: 'dashboard', component: ShowcaseDashboardPageComponent, data: { navigation: false } },
```